### PR TITLE
📖  Reorganize Getting Started Page

### PR DESCRIPTION
**Pull request by MikeSpreitzer**
## Summary
This PR reorganizes the Getting Started page a bit, to follow @KPRoche's outline shown in https://github.com/kubestellar/kubestellar/pull/2584#issuecomment-2476820172 . This is good because it puts the fast path first without reducing what can be learned from the whole page.

This version also draws on the version in release 0.25.0 (https://docs.kubestellar.io/release-0.25.0/direct/get-started/).

Preview at https://mikespreitzer.github.io/kcp-edge-mc/doc-demo-script/direct/get-started/

## Related issue(s)

Fixes #

### CODE REVIEW

**Comment by MikeSpreitzer** [comment]
/cc @KPRoche
/cc @francostellari
/cc @clubanderson
@katamyra

File: docs/content/direct/get-started.md [hunk 1]
```diff
@@ -137,6 +133,30 @@ wec2_context=$wec2_name
 label_query_both=location-group=edge
 label_query_one=name=cluster1
 ```
+
+## Exercise KubeStellar
+
+Now that your system is running, you can try some example scenarios
+
+1. Define the needed shell variables, using either the settings output as the script completes or the settings shown just above from the step-by-step instructions). Their meanings are defined [at the start of the example scenarios document](example-scenarios.md#assumptions-and-variables).
```
Comment (KPRoche) [inline docs/content/direct/get-started.md:141]: Minor typo here: there is an orphan closing parenthesis after "instructions"

**Comment by MikeSpreitzer** [comment]
The force-push to cb14584e0 fixes the extra paren noted above.

File: docs/content/direct/get-started.md (hunk 1 above)
Comment (MikeSpreitzer) [reply docs/content/direct/get-started.md]: Thanks. Fixed.

**Pull request merged**